cloud/docker/docker_image.py

```diff
@@ -371,12 +371,6 @@ def push_image(self, name, tag=None):
         registry, repo_name = auth.resolve_repository_name(repository)
 
         if re.search('/', repository):
-            if registry:
```

Comment:
I was thinking on 'found_credentials = True|False' , you can then use when the connection fails with 'unauthorized' and then issue this fail vs the other one that indicates that the repo is missing.